health.go

```diff
@@ -9,6 +9,7 @@ import (
 	"time"
 	"bufio"
 	"os"
+	"github.com/kkdai/maglev"
```

Comment:
Hm, I didn't see the license on the larger project.  More worryingly however, some of the code from kkdai's maglev implementation seems to be copied from https://github.com/dgryski/go-maglev which is also unlicensed.  (At least the test code is obviously derivative, and the populate function seems to be derivative too.)  I think in order to avoid any copyright issues it would be best to write our own implementation, which I can do this weekend.